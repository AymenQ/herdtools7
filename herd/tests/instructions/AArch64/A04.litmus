AArch64 A4
(* Tests load immediate, no offset, from location *)
{ 0:X1=x; uint64_t x }

P0;
  LDR X0, [X1];

exists (0:X0=0)
