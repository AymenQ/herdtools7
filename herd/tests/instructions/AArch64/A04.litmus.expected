Test A4 Allowed
States 1
0:X0=0;
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists (0:X0=0)
Observation A4 Always 1 0
Hash=8940fd2b06207ee57113ad068cb81bce

